index.js

```diff
@@ -202,8 +200,7 @@ module.exports = (request, options) => {
                 targetFilePath = value;
               else
                 for(const [key2, value2] of Object.entries(value))
-                  if(key2 === "import"
-                  || key2 === "require"
+                  if(conditions.indexOf(key2) !== -1
```

Comment:
Use `includes()` here. Also, it should not check for `default` again, it's an error.